blog/2023-07-07-hydra.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Hydra Team Update
+slug: 2023-07-07-hydra
+authors: [ffakenz, v0d1ch]
+tags: [hydra]
+hide_table_of_contents: false
+---
+
+## High-level summary
+
+This week the Hydra team accomplished some nice progress. They secured the
+network layer further by implementing authentication of the messages between
+the peers in the Head protocol. In the process they also managed to separate
+HeartBeat messages from the protocol ones which somewhat improved the quality
+of code in Hydra. The team also finished work related to sending only
+transaction ids in ReqSn messages and fixed an issue in our smoke-tests. They
+also published multiple benchmarks on their website and identified a bug in the
+signature validation and issued a pull request to
+[fix](https://github.com/input-output-hk/hydra-ghsa-c8qp-cv4h-vcc4/pull/1)
+this.
```

Comment:
I suggest to not put more detail in the summary than in the bullets below. Concretely: would move this sentence to the detail list below